documentation/theme/main.html

```diff
@@ -1,23 +1,39 @@
-{% extends "base.html" %} 
+{% extends "base.html" %}
 
 {% block extrahead %}
-<!-- TODO: should shape these up and should cater to specific page -->
-<meta name="description" content="Documentation for the SDK to build Coda Packs." />
+
+{% set title = config.site_name %}
+{% if page.toc|first is defined %}
```

Comment:
where is the template language defined? looks like jinja2 template.

Reply:
Ya, mkdocs uses Jinja2: https://www.mkdocs.org/dev-guide/themes/#creating-a-custom-theme

Reply:
thanks for the pointer!